lib/api/apiUtils/bucket/bucketCreation.js

```diff
@@ -172,8 +183,6 @@ export function createBucket(authInfo, bucketName, headers,
 
     if (locationConstraint !== undefined) {
         bucket.setLocationConstraint(locationConstraint);
-    } else if (usEastBehavior) {
-        bucket.setLocationConstraint('us-east-1');
```

Comment:
After creating a bucket with undefined locationConstraint 
`./s3curl.pl --id aws --contentType application/x-www-form-urlencoded --createBucket -- https://s3.amazonaws.com/cocototo -v `, it actually set the location to us-east-1.
Note: we only keep it for the old config since usEastBehavior has been removed in the new one.
Does it make sense @LaurenSpiegel ?